front/src/views/Boxes/components/BoxesActionsAndTable.tsx

```diff
@@ -207,6 +211,17 @@ function BoxesActionsAndTable({
     deleteBoxes(selectedBoxes.map((box) => box.values as IBoxBasicFields));
   }, [deleteBoxes, selectedBoxes]);
 
+  // Export Boxes
+  const exportBoxData = () => {
```

Comment:
@jlin95 would be great to move this and the corresponding state to a subcomponent. not sure btw why you need the exportData state and cannot transform the data once the click happens and then directly send it to the CSVLink component.